test.sh

```diff
@@ -8,6 +8,25 @@
 
 set -e
 
+
+if [[ -f "./manifest.sh" ]]; then
```

Comment:
`manifest.sh` will no longer be used for python https://github.com/sclorg/s2i-python-container/pull/539. If the new solution works in python, the plan is to use it in all distgen-generated container repositories.
We can, of course, use this solution till then, but let's count with a fact, that this is temporary.

Reply:
Ok, make sense to me, but we have to rewrite it also for `varnish-container` and `postgresql-container`.

Reply:
Or we can check if `manifest.sh` or `manifest.yaml` exists.

Reply:
There can also be files named like [manifest-minimal.yml](https://github.com/sclorg/s2i-python-container/pull/539/files#diff-4e820cda54f76392ca641e6eb4b84a36ae0d5c50befa3444f12fc08a3ac70805).

Reply:
But well, yes, we could check for `manifest*.yaml`

Reply:
Also the Makefile rule is called `generate-new` here and not `generate-all`.

Reply:
I would wait with these changes until the https://github.com/sclorg/s2i-python-container/pull/539 is merged.

Reply:
> Also the Makefile rule is called `generate-new` here and not `generate-all`.

Sorry I don't see `generate-new` I see only `generate-all` https://github.com/sclorg/container-common-scripts/blob/master/common.mk#L147

Reply:
I meant in the new proposal from Lumir.

Reply:
We agreed to ignore generate-new and manifest.yml for now.